contracts/contracts/strategies/NativeStaking/ValidatorRegistrator.sol

```diff
@@ -267,66 +267,87 @@ abstract contract ValidatorRegistrator is Governable, Pausable {
         );
     }
 
-    // slither-disable-end reentrancy-no-eth
-
-    /// @notice Exit a validator from the Beacon chain.
+    /// @notice Exit validators from the Beacon chain.
     /// The staked ETH will eventually swept to this native staking strategy.
     /// Only the registrator can call this function.
-    /// @param publicKey The public key of the validator
+    /// @param publicKeys List of SSV validator public keys
     /// @param operatorIds The operator IDs of the SSV Cluster
     // slither-disable-start reentrancy-no-eth
-    function exitSsvValidator(
-        bytes calldata publicKey,
+    function exitSsvValidators(
+        bytes[] calldata publicKeys,
         uint64[] calldata operatorIds
-    ) external onlyRegistrator whenNotPaused {
-        bytes32 pubKeyHash = keccak256(publicKey);
-        VALIDATOR_STATE currentState = validatorsStates[pubKeyHash];
-        require(currentState == VALIDATOR_STATE.STAKED, "Validator not staked");
+    ) external onlyRegistrator whenNotPaused nonReentrant {
+        ISSVNetwork(SSV_NETWORK).bulkExitValidator(publicKeys, operatorIds);
 
-        ISSVNetwork(SSV_NETWORK).exitValidator(publicKey, operatorIds);
+        bytes32 pubKeyHash;
```

Comment:
nit: probably no reason to declare this outside of the loop

Reply:
The reason is gas although it's only a very small gain. 
Iterating over 3 validators
Outside the loop: 109,504
Inside the loop: 109,511